.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,47 @@
+**Is it a ?**
+
+- [ ] Bug Report
+
+    **Actual Behaviour**
+
+    Please state here what is currently happening.
+
+    **Expected Behaviour**
+
+    State here what the feature should enable the user to do.
+
+    **Steps to reproduce it**
+
+    Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.
+
+    **LogCat for the issue**
+
+    Provide logs for the crash here
+
+    **Screenshots of the issue**
+
+    Where-ever possible attach a screenshot of the issue.
+    
+    **Android version and Phone Model**
+
+    Add the Android version and the phone model on which the issue was encountered.
+
+- [ ] Feature Request
+    
+    **Feature Description**
+
+    Please describe the feature you want to add to the project.
+
+    **Importance/Application**
+
+    Please state the importance/application of this new feature.
```

Comment:
I think this can be omitted. The more we require people to write, the less they wan to :smile: Feature description is adequate